ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/requesthandler/internalresponse/EthSignResultProvider.java

```diff
@@ -75,18 +72,12 @@ public JsonRpcBody getBody(final JsonRpcRequest request) {
               Bytes32.leftPad(Bytes.wrap(ByteUtils.bigIntegerToBytes(signature.getR()))),
               Bytes32.leftPad(Bytes.wrap(ByteUtils.bigIntegerToBytes(signature.getS()))),
               Bytes.wrap(ByteUtils.bigIntegerToBytes(signature.getV())));
-      final JsonRpcSuccessResponse response =
-          new JsonRpcSuccessResponse(
-              request.getId(), Numeric.toHexString(outputSignature.toArray()));
-      return new JsonRpcBody(Json.encodeToBuffer(response));
+      return Numeric.toHexString(outputSignature.toArray());
     } catch (final ClassCastException e) {
```

Comment:
Now that the general exception isn't being handled, can the try/catch for the castClassException be narrowed to be just around the casting to make it clearer we aren't handling the exception